version/version_test.go

```diff
@@ -41,13 +41,16 @@ func TestVersionCheck(t *testing.T) {
 		wantError       bool
 	}{{
 		name:          "greater version (patch)",
-		actualVersion: &testVersioner{version: "v1.20.2"},
+		actualVersion: &testVersioner{version: "v1.20.0"},
 	}, {
 		name:          "greater version (patch), no v",
-		actualVersion: &testVersioner{version: "1.20.2"},
+		actualVersion: &testVersioner{version: "1.20.0"},
 	}, {
 		name:          "greater version (patch), pre-release",
-		actualVersion: &testVersioner{version: "1.20.2-kpn-065dce"},
+		actualVersion: &testVersioner{version: "1.20.0-kpn-065dce"},
```

Comment:
Changing `1.20.2` to `1.20.0` as comparing `1.20.2-kpn-065dce` with `1.20.0` could miss expected check.
This change is not related to the original PR actually so I can send another PR if I should separate it.